recipes-aos/aos-messageproxy/aos-messageproxy_git.bb

```diff
@@ -1,14 +1,11 @@
 DESCRIPTION = "AOS Message Proxy"
-
-GO_IMPORT = "github.com/aosedge/aos_message_proxy"
-
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
 
 BRANCH = "develop"
 SRCREV = "${AUTOREV}"
 
-SRC_URI = "git://${GO_IMPORT}.git;branch=${BRANCH};protocol=https"
+SRC_URI = "git://github.com/MykolaSolyanko/aos_core_mp_cpp.git;protocol=https;branch=${BRANCH}"
```

Comment:
put aosedge repo

Reply:
will do when PR for message-proxy will be merged

Reply:
Done